packages/eslint-config-mc-app/package.json

```diff
@@ -22,8 +22,8 @@
   "module": "./index.js",
   "peerDependencies": {
     "babel-eslint": "10.x",
-    "eslint": "5.x",
-    "eslint-config-airbnb-base": "13.x",
+    "eslint": "^5.16.0 || ^6.1.0",
```

Comment:
```suggestion
    "eslint": "6.x",
```

Reply:
You just want to be harsh? We could work with 5.16 but you don't want to?

Reply:
I don't understand, you said yourself:

> However v5 and v14 won't work together

https://github.com/commercetools/merchant-center-application-kit/pull/911#discussion_r312810940

Reply:
Ah, no to my understanding:

1. v14 works with v5 and v6
2. v13 works only with v5

So I "decided" (rather wanted to suggest) to remove support for the `-base` of v13 which as a result allows us to keep supporting v5 and v6. Hope that's less confusing now :)

Reply:
I see, sure ok fine by me then